satpy/etc/readers/nwcsaf-geo.yaml

```diff
@@ -1,7 +1,7 @@
 reader:
   description: NetCDF4 reader for the NWCSAF MSG Seviri 2016/2018 format
   name: nwcsaf-geo
-  sensors: [seviri]
+  sensors: [seviri, abi, ahi]
```

Comment:
Would it be premature to also add `fci` here?  NWCSAF are already working on testing it and geophysically realistic FCI test data is coming soon, so in theory it should soon be possible to produce test FCI NWCSAF products, although of course the real products are still a while away.

Reply:
Should be done in a separate PR, see above.